### PR TITLE
Added app-veyor badge

### PR DESCRIPTION
**Pull request by soywiz**
Hopefully, this will also trigger appveyor (I have added the webhook already)

### CODE REVIEW

**Comment by soywiz** [comment]
@fxzjshm I have tried several things to configure appveyor (in settings):
https://ci.appveyor.com/project/soywiz/jtransc/build/1.0.298
https://ci.appveyor.com/project/soywiz/jtransc/build/1.0.296
It seems that it is ignoring `appveyor.yml`. Do you know how to configure it?

**Comment by fxzjshm** [comment]
It seems that you forgot to merge master?

**Comment by soywiz** [comment]
Woops! :D you are right. Going to try.

**Comment by soywiz** [comment]
Looks almost good. It seems that test fails due to some kind of encoding error on apoveyor. Ill check this evening

**Comment by fxzjshm** [comment]
😁

File: appveyor.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@ environment:
   LINKCMD64: C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\amd64\link.exe
   LIB: C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\lib;C:\Program Files (x86)\Windows Kits\10\Lib\10.0.14393.0\um\x64;C:\ProgramData\chocolatey\lib\dmd\tools\dmd2\windows\lib64;C:\Program Files (x86)\Windows Kits\10\Lib\10.0.14393.0\ucrt\x64;
 install:
+  - cmd: chcp # Página de códigos activa: 850
```
Comment (intrigus) [inline appveyor.yml]: Could this be in English please :)

Comment (soywiz) [inline appveyor.yml]: Yes. Removed. It was for reference and forgot removing it

**Pull request merged**